src/doc/src/guide/continuous-integration.md

```diff
@@ -21,6 +21,42 @@ will not fail your overall build. Please see the [Travis CI Rust
 documentation](https://docs.travis-ci.com/user/languages/rust/) for more
 information.
 
+### GitHub Actions
+
+To test your package on GitHub Actions, here is a sample `.github/workflows/ci.yml` file:
+
+```yaml
+name: Cargo Build & Test
+
+on:
+  push:
+
+jobs:
+  build_and_test:
+    name: Rust project - latest
+    runs-on: ${{ matrix.operating-system }}
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        toolchain:
+          - stable
+          - beta
+          - nightly
+        operating-system:
+          - "ubuntu-latest"
```

Comment:
If we provide an example in the documentation I think we should make it somewhat terse and not have too many extra bells and whistles. To that extent I think this would be removed and folded directly into the `runs-on`.

Additionally I think that `continue-on-error` and `fail-fast` would be removed because we don't necessarily have a great reason for deviating from the GitHub Actions defaults.